libraries/server/auth/src/main/java/com/chesire/nekome/auth/api/AuthEntity.kt

```diff
@@ -0,0 +1,7 @@
+package com.chesire.nekome.auth.api
+
+// Currently not needed - will be used in a refactor #387
+/**
+ * Provides an entity for the Auth module.
+ */
+class AuthEntity
```

Comment:
Maybe an AuthDTO instead

Reply:
Or an AuthDomain instead? So DTO is for App->Server, and Domains are from server layer -> repo/app layer?